src/material/schematics/ng-update/migrations/theming-api-v12/migration.ts

```diff
@@ -207,6 +207,12 @@ function renameSymbols(content: string,
 /** Inserts an `@use` statement in a string. */
 function insertUseStatement(content: string, importPath: string, importsToIgnore: string[],
                             namespace: string): string {
+  // If the content already has the `@use` import, we don't need to add anything.
+  const alreadyImportedPattern = new RegExp(`@(import|use) +['"]${importPath}['"]`, 'g');
```

Comment:
I think that this should be just `@use` not `@(import|use)`.

Reply:
Yep, good catch